src/index.js

```diff
@@ -154,23 +156,32 @@ function performCheck(input: HTMLInputElement, body: FormData, url: string): Pro
   requests.set(input, xhr)
 
   xhr.open('POST', url, true)
-  xhr.setRequestHeader('Accept', 'text/html; fragment')
+  xhr.setRequestHeader('Accept', 'application/json, text/plain;q=0.9')
```

Comment:
WDYT about not setting an accept header and should just accept anything, for the success case?

Reply:
Sounds good